.github/workflows/release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Draft release
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: write # For creating releases.
+  discussions: write #  For creating a discussion.
+
+# Cancel in-progress runs for pull requests when developers push
+# additional changes
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  draft:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Download artifacts
+        uses: dawidd6/action-download-artifact@v2
+        id: download
+        with:
+          branch: main
+          workflow: ci.yaml
+          workflow_conclusion: completed
+          check_artifacts: true
+          name: release-packages
+          path: ./release-packages
+
+      - uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          discussion_category_name: "📣 Announcements"
+          files: ./release-packages/*
+          body_path: ci/build/release-notes.txt
```

Comment:
We can probably remove this since we will just overwrite this when copying from the changelog right?  I wonder if we can somehow get fancy and automatically extract the last block from the changelog.

Reply:
Yeah that's true. I'll delete `release-notes.txt` too since it's already in the changelog.

> I wonder if we can somehow get fancy and automatically extract the last block from the changelog.

Haha that would be awesome! Probably some script step before this one that parses `CHANGELOG` -> grabs relevant block -> outputs to file -> we use that.